tests/testthat/_snaps/survival-workflow-set.md

```diff
@@ -0,0 +1,11 @@
+# resampling survival models with static metric
+
+    Code
+      wflow_set_fit_stc_eval <- workflow_map(wflow_set, "fit_resamples", seed = 2193,
+        resamples = sim_rs, metrics = stc_mtrc, eval_time = time_points, control = ctrl)
+    Condition
+      Warning in `tune::fit_resamples()`:
+      Evaluation times are only required when dynamic or integrated metrics are used (and will be ignored here).
+      Warning in `tune::fit_resamples()`:
+      Evaluation times are only required when dynamic or integrated metrics are used (and will be ignored here).
```

Comment:
Note the duplicate warnings.👀

This is a little bit different than the similar output we saw for finetune, as users can vary `options` such that different workflows would experience different issues (so we can't just run the newly exported tune checking functions for one workflow and assume it's output applies elsewhere). Because of this, workflowsets doesn't deduplicate errors/warnings across tuning results.

Reply:
Thanks for the context here